app/views/candidates/placement_requests/cancellations/_show_booking.html.erb

```diff
@@ -0,0 +1,40 @@
+<% self.page_title = 'Cancelled booking' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
+        Your school experience booking has been cancelled
+      </h1>
+    </div>
+
+    <h2 class="govuk-heading-m">What happens next</h2>
+    <p>
+    Your school experience booking on <%= cancellation.booking_date %> at <%= cancellation.school_name %> has been cancelled.
+    </p>
+
+    <h2 class="govuk-heading-s">The school will be notified about this cancellation.</h2>
+    <p>
+      We'll also send you an email to confirm you've cancelled your booking.
+    </p>
+
+    <h2 class="govuk-heading-m">Get advice about teaching</h2>
+    <p>
+      If you’d like to find out more about teaching or teacher training call our Get Into Teaching service on Freephone 0800 389 2500.
+    </p>
+    <p>
+      You can also chat to a Get Into Teaching Line adviser using our
+      <%= link_to 'online chat service', 'https://ta-chat.education.gov.uk/chat/chatstart.aspx?domain=www.education.gov.uk&department=GetIntoTeaching%27,%27new_win%27,%27width=0,height=0%27);return&SID=0' %> between 8am and 8pm, Monday to Friday.
+    </p>
+
+
+    <% if false %>
```

Comment:
Can this change to check for phase 4 - we should be rolling out then. I've gated the dashboard at phase 4 - if we move it to a later phase at least I'll find it when I grep for phase checks